packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -588,4 +604,8 @@ export class BlobManager {
         }
         return blobs;
     }
+
+    public hasPendingBlobs(): boolean {
```

Comment:
nit: This could be a getter, and I think it would be better to put it below the constructor near other getters (pendingOfflineUploads/hasPendingOfflineUploads)